rclpy/executors/setup.py

```diff
@@ -0,0 +1,37 @@
+from ament_python.data_files import get_data_files
+from ament_python.script_dir import install_scripts_to_libexec
+from setuptools import setup
+
+package_name = 'examples_rclpy_executors'
+data_files = get_data_files(package_name)
+install_scripts_to_libexec(package_name)
```

Comment:
The currently pending PRs haven't been finally reviewed but we talked about the proposed change recently and I expect them to get merged as is rather soon.